backend/inspirehep/records/api/literature.py

```diff
@@ -174,7 +174,7 @@ def update(self, data, disable_orcid_push=False, *args, **kwargs):
     def get_modified_authors(self):
         previous_authors = self._previous_version.get("authors", [])
         previous_authors_by_uuid = {
-            author["uuid"]: author for author in previous_authors
+            author["uuid"]: author for author in previous_authors if "uuid" in author
```

Comment:
I don't think we should do that as it hides issues. We need to fix the underlying problem.